docs/getting-started/cluster-setup.md

```diff
@@ -0,0 +1,232 @@
+```yaml
+---
+login:
+    # A private key uses for signing jwt tokens. Can be anything
+    jwtPrivateKey: THIS-IS-AN-INSECURE-KEY
+    # The client id used for OAuth with github. Look up GitHub OAuth for details
+    # https://developer.github.com/v3/oauth/
+    oauthClientId: YOU-PROBABLY-WANT-SOMETHING-HERE
+    # The client secret used for OAuth with github
+    oauthClientSecret: AGAIN-SOMETHING-HERE-IS-USEFUL
+    # A password used for encrypting session, and OAuth data.
+    # **Needs to be minimum 32 characters**
+    password: WOW-ANOTHER-INSECURE-PASSWORD!!!
+    # A flag to set if the server is running over https.
+    # Used as a flag for the OAuth flow
+    https: false
+
+httpd:
+    port: 8080
+
+datastore:
+    plugin: dynamodb
+    imdb:
+        # File to read/write the database to
+        filename: ./database.json
+    dynamodb:
+        # AWS Access Key ID
+        accessKeyId: WHAT-A-LEGITIMATE-LOOKING-KEY-ID
+        # AWS Secret Access Key
+        secretAccessKey: TOTALLY-REAL-LOOKING-AWS-KEYS
+
+executor:
+    plugin: k8s
+    k8s:
+        # The host or IP of the kubernetes cluster
+        host: kubernetes
+        # The jwt token used for authenticating kubernetes requests
+        token: NOT-A-REAL-JWT-TOKEN
+
+webhooks:
+    github:
+        # Secret to add to GitHub webhooks so that we can validate them
+        secret: SUPER-SECRET-SIGNING-THING
+```
+
+#### Kubernetes secrets
+Kubectl can be used to see your [Kubernetes secrets](http://kubernetes.io/docs/user-guide/secrets/walkthrough/).
+To get your k8s `host`, run:
+```bash
+$ kubectl cluster-info
+
+Kubernetes master is running at https://55.555.55.55
+```
+
+To get the k8s `<DEFAULT_TOKEN_NAME>`, run:
+```bash
+$ kubectl get secrets
+
+NAME                      TYPE                                  DATA      AGE
+datastore-dynamodb        Opaque                                2         19d
+db-user-pass              Opaque                                2         49d
+default-token-abc55       kubernetes.io/service-account-token   3         50d
+git-token                 Opaque                                1         42d
+mysecret                  Opaque                                2         49d
+npm-token                 Opaque                                1         42d
+plugin-login-secrets      Opaque                                4         44d
+screwdriver-job-secrets   Opaque                                1         43d
+```
+
+The `<DEFAULT_TOKEN_NAME>` will be listed under `Name` when the `Type` is `kubernetes.io/service-account-token`.
+
+To get your k8s `token`, run:
+```bash
+$ kubectl describe secrets <DEFAULT_TOKEN_NAME>  # default-token-abc55 in this example
+```
+
+#### DynamoDB secrets
+To get your `accessKeyId` and `secretAccessKey`, navigate to [IAM](https://console.aws.amazon.com/iam)
+in your AWS console. Click on Users, then your `<USERNAME>`. Select the `Security Credentials` tab and click
```

Comment:
We shouldn't recommend that people use their own IAM user for this. We should recommend that people setup a dedicated user with the right permissions.